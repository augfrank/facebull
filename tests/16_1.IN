M0   C0   C2  2702
M1   C0   C6  2205
M2   C0   C15  2254
M3   C1   C0  2760
M4   C2   C9  2553
M5   C3   C1  2288
M6   C3   C8  2370
M7   C3   C10  2037
M8   C4   C2  2255
M9   C5   C1  2275
M10   C5   C6  2743
M11   C5   C8  2671
M12   C6   C11  2012
M13   C6   C13  2762
M14   C7   C0  2364
M15   C7   C5  2805
M16   C7   C11  2431
M17   C7   C12  2676
M18   C8   C2  2437
M19   C8   C7  2652
M20   C8   C10  2492
M21   C8   C12  2745
M22   C9   C8  2196
M23   C9   C14  2169
M24   C9   C15  2363
M25   C10   C1  2428
M26   C10   C11  2481
M27   C11   C4  2989
M28   C11   C7  2076
M29   C12   C6  2276
M30   C12   C10  2874
M31   C13   C0  2369
M32   C13   C3  2402
M33   C13   C12  2025
M34   C13   C15  2643
M35   C14   C3  2641
M36   C14   C8  2253
M37   C14   C13  2148
M38   C15   C7  2755
M39   C15   C13  2885
