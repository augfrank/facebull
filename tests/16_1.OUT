39259
2 3 4 7 8 11 12 15 21 23 25 27 29 32 37 38
